libfreerdp/common/assistance.c

```diff
@@ -704,7 +704,7 @@ static BOOL freerdp_assistance_decrypt2(rdpAssistanceFile* file, const char* pas
 BYTE* freerdp_assistance_hex_string_to_bin(const void* raw, size_t* size)
 {
 	BYTE* buffer = NULL;
-	size_t length, rc;
+	size_t length = 0, rc = 0;
```

Comment:
false positive, better move down below as `const size_t length = strlen...`, same for rc

Reply:
Fixed.